### PR TITLE
feat(wrapStory): allow wrapping stories with additional wrappers

### PR DESCRIPTION
**Pull request by arahansen**
Might be overkill to add the default wrapStory.js to config. The rationale was that we have a "default config" that mirrors the sort of setup we recommend for users to build.

Alternatively, we can do something different inside preval to conditionally return an identity function if the `wrapStory` option is not set. Although that might be annoying since we are doing additional parsing of the config value given to us by prepending the root directory to the path provided.

### CODE REVIEW

File: config/wrapStory.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default function(story) { return story }
```
Comment (obartra) [inline config/wrapStory.js]: you could just use `_.identity` instead of adding a new file

Comment (arahansen) [reply config/wrapStory.js]: Thats what I meant by this line:
> Alternatively, we can do something different inside preval to conditionally return an identity function if the wrapStory option is not set

Comment (obartra) [reply config/wrapStory.js]: Yeah all I'm saying is I agree this seems overkill for now but I don't feel too strongly about it

**Comment by arahansen** [comment]
@obartra this should be ready to merge now, when you get the chance.

**Pull request merged**